haiku-apps/stopwatch/stopwatch-1.0.0.recipe

```diff
@@ -0,0 +1,40 @@
+SUMMARY="A simple stopwatch application"
+DESCRIPTION="This stopwatch is very simple to use and allows a user to start, stop and reset the clock."
+HOMEPAGE="https://github.com/dcatt-haiku/StopWatch"
+COPYRIGHT="2016 DCatt"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/dcatt-haiku/StopWatch/archive/v1.0.0.tar.gz"
```

Comment:
You could change this to "https://github.com/dcatt-haiku/StopWatch/archive/v$portVersion.tar.gz" to make updates to future release even easier.